subgraph-js/package.json

```diff
@@ -23,7 +23,10 @@
   "publishConfig": {
     "access": "public"
   },
+  "dependencies": {
+    "apollo-server-types": "^0.9.0 || ^3.0.0"
```

Comment:
Is this a stray addition?

Reply:
32384c0

Reply:
I had to add it to get it to work, but I'm not sure why.